master/buildbot/db/model.py

```diff
@@ -104,6 +104,7 @@ class Model(base.DBConnectorComponent):
     # claim is made by the master referenced by masterid.
     buildrequest_claims = sautils.Table(
         'buildrequest_claims', metadata,
+        sa.Column('dummy', sa.Integer, primary_key=True, autoincrement=True),
         sa.Column('brid', sa.Integer, sa.ForeignKey('buildrequests.id'),
```

Comment:
The buildrequest_claims table is weird. If you really want the brid field to be the primary key, then having that table is not necessary since all fields on it could be moved to the buildrequests table.

brid + masterid doesn't make sense to be a composite primary key as masterid can be NULL.